src/ArborX_LinearBVH.hpp

```diff
@@ -24,9 +24,11 @@
 #include <ArborX_DetailsSortUtils.hpp>
 #include <ArborX_DetailsTreeConstruction.hpp>
 #include <ArborX_DetailsTreeTraversal.hpp>
+#include <ArborX_Polytope.hpp>
 #include <ArborX_TraversalPolicy.hpp>
 
 #include <Kokkos_Core.hpp>
+#include <Kokkos_HostSpace.hpp>
```

Comment:
Why? Even if you need it, wouldn't it be included as part of `Kokkos_Core.hpp`?

Reply:
Ignore.  Was added by static analyzer and I do not plan on committing this